extensions/amp-sidebar/1.0/test/test-component.js

```diff
@@ -741,68 +745,50 @@ describes.sandboxed('Sidebar preact component', {}, (env) => {
               </ul>
             </SidebarToolbar>
           </Sidebar>
-          <button id="open" onClick={() => ref.current.open()}></button>
-          <button id="close" onClick={() => ref.current.close()}></button>
         </>
       );
 
-      const openButton = wrapper.find('#open');
-      const closeButton = wrapper.find('#close');
-
-      // verify sidebar is closed
-      let sidebarElement = wrapper.find(Sidebar).getDOMNode();
-      expect(isOpened(sidebarElement)).to.be.false;
-
-      // Toolbar target should have children
       expect(target.hasChildNodes()).to.be.true;
+      expect(target.childElementCount).to.equal(2);
 
-      // click to open the sidebar
-      openButton.simulate('click');
-
-      // verify sidebar is opened and toolbar has children
-      sidebarElement = wrapper.find(Sidebar).getDOMNode();
-      expect(isOpened(sidebarElement)).to.be.true;
-      expect(target.hasChildNodes()).to.be.true;
+      wrapper.unmount();
+      expect(target.hasChildNodes()).to.be.false;
+    });
 
-      // click to close the sidebar
-      closeButton.simulate('click');
+    it('toolbar should sanitize an invalid media query', () => {
```

Comment:
The diff looks a little odd in my gitHub window.  Not sure if it's the case for other reviewers.  I renamed a few tests above and added this test and the one below.